_includes/contact_form.html

```diff
@@ -1,3 +1,10 @@
+{% comment %}
+
+include:
+- common_applications: array of strings (translation keys)
+- all_applications: array of (translation keys)
```

Comment:
Do we need this to be listed out in page data and given to the partial, or could we just get them from `settings.contact_page.support_request_form.agencies` ?

```erb
{% for kv in settings.contact_page.support_request_form.agencies %}
  {% assign value = kv[0] %}
  {% assign agency = kv[1] %}
{% endfor %}
```

**Edit:** I notice "Other" is lumped in there as well, but maybe we can just move it up a level out of agencies and reference it directly. Probably more appropriate that way anyways, since "Other" isn't an agency.

Reply:
Yeah, I can remove the all_applications, I do think the common one is a nice way to minimize a little bit of duplication